widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -335,10 +335,18 @@ public void emitState(CallState callState) {
                     muteButton.setEnabled(callState.isAudioCall() || callState.is2WayVideoCall());
                     speakerButton.setEnabled(callState.isAudioCall() || callState.is2WayVideoCall());
                     videoButton.setEnabled(callState.is2WayVideoCall());
-                    setButtonActivated(videoButton, theme.getIconCallVideoOn(),
-                            theme.getIconCallVideoOff(), callState.hasVideo);
-                    setButtonActivated(muteButton, theme.getIconCallAudioOn(),
-                            theme.getIconCallAudioOff(), callState.isMuted);
+                    setButtonActivated(
+                            videoButton,
+                            theme.getIconCallVideoOn(),     // activated
```

Comment:
I really don't like those short debuggish comments. Remove them or make them more descriptive.

Reply:
agree with @jakob-mc  comment. Let's make it more descriptive.

Reply:
reverted the videobutton changes - made mute button comment more descriptive